pkgs/applications/virtualization/openstack/openstackclient/default.nix

```diff
@@ -0,0 +1,43 @@
+{ pkgs, stdenv, python2Packages }:
+
+let
+  python = { mkDerivation = pkgs.python2Packages.buildPythonPackage; };
+  generated = import ./requirements_generated.nix { inherit pkgs python; } ;
+in
+
+python2Packages.buildPythonApplication rec {
+  pname = "python-openstackclient";
+  version = "3.11.0";
+  name = "${pname}-${version}";
+
+  src = python2Packages.fetchPypi {
+    inherit pname version;
+    sha256 = "79fc71a0234f212fc4cccb53b64763be7753e02c1666957a6210b16e8c68c2f6";
+  };
+
+  # We don't have build requires
+  doCheck = false;
+
+  propagatedBuildInputs = [
```

Comment:
hmm, I have to use `fix` instead of `fix'` but I still don't understand (ok, I don't try too much to understand....).